### ISSUE TITLE
Browser dependencies moved to bower_components folder

### ISSUE THREAD

**Issue by alelb** [fabbricadigitale/paper-chip#6]
Due to `web-component-tester` issue Polymer/web-component-tester/issues/488, browser dependencies are moved to `bower_components` folder waiting for a solution.

**Comment by leodido** [comment]
A better (temporary) workaround would be to create a symbolic link.

Reasoning:
1. No additional deps
2. Plain node - ie., `fs.symlink()`

**Comment by leogr** [comment]
I agree to leave it as is. We have just to remember to [revert the folder](https://github.com/fabbricadigitale/paper-chip/commit/d2ac8f4c7b930e64cced27d7a362c2ea26331514#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L32) when `wct` is fixed (or `bower` is gone away definitively).

**Comment by leodido** [comment]
Any news on wct about this?
